### PR TITLE
Add .gitattributes

### PR DESCRIPTION
**Pull request by nitesh31mishra**
Added the .gitattributes file. It handles EOL characters on repository files for Mac and Windows OS

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+* text=auto
+
+# Force batch scripts to always use CRLF line endings so that if a repo is accessed
+# in Windows via a file share from Linux, the scripts will work.
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
+
+# Force bash scripts to always use LF line endings so that if a repo is accessed
+# in Unix via a file share from Windows, the scripts will work.
+*.sh text eol=lf
```
Comment (agupta15k) [inline .gitattributes]: No newline character at end of file. Please add a new line character and update the PR.

**Comment by agupta15k** [comment]
https://github.com/all-contributors please add @Arunsp2000  for infrastructure, tests and code

File: .gitattributes (hunk 1 above)
Comment (nitesh31mishra) [reply .gitattributes]: Updated.

**Review by agupta15k — COMMENTED** [review]
Approved

**Pull request merged**